docs/ldml/tr35-keyboards.md

```diff
@@ -122,11 +122,11 @@ The Unicode Standard and related technologies such as CLDR have dramatically imp
 
 <!-- The CLDR keyboard format provides for the communication of keyboard mapping data between different modules, and the comparison of data across different vendors and platforms. The standardized identifier for keyboards can be used to communicate, internally or externally, a request for a particular keyboard mapping that is to be used to transform either text or keystrokes. The corresponding data can then be used to perform the requested actions.
 
-For example, a web-based virtual keyboard may transform text in the following way. Suppose the user types a key that produces a "W" on a qwerty keyboard. A web-based tool using an azerty virtual keyboard can map that text ("W") to the text that would have resulted from typing a key on an azerty keyboard, by transforming "W" to "Z". Such transforms are in fact performed in existing web applications.
+For example, a web-based virtual keyboard may transform text in the following way. Suppose the user types a key that produces a "W" on a QWERTY keyboard. A web-based tool using an AZERTY virtual keyboard can map that text ("W") to the text that would have resulted from typing a key on an AZERTY keyboard, by transforming "W" to "Z". Such transforms are in fact performed in existing web applications.
 
 The data can also be used in analysis of the capabilities of different keyboards. It also allows better interoperability by making it easier for keyboard designers to see which characters are generally supported on keyboards for given languages.
 
-To illustrate this specification, here is an abridged layout representing the English US 101 keyboard on the Mac OSX operating system (with an inserted long-press example). For more complete examples, and information collected about keyboards, see keyboard data in XML.
+To illustrate this specification, here is an abridged layout representing the English US 101 keyboard on the Mac OS X operating system (with an inserted long-press example). For more complete examples, and information collected about keyboards, see keyboard data in XML.
```

Comment:
```suggestion
To illustrate this specification, here is an abridged layout representing the English US 101 keyboard on the macOS operating system (with an inserted long-press example). For more complete examples, and information collected about keyboards, see keyboard data in XML.
```

They keep changing the name of the operating system. I believe this is current, per Apple's website: https://www.apple.com/macos/monterey/